src/screens/profile/styles.ts

```diff
@@ -1,9 +1,31 @@
 import { StyleSheet } from 'react-native';
 
+import { AppColor } from 'src/common/enums';
+
 const styles = StyleSheet.create({
   fullHeight: {
     height: '100%',
   },
+  container: {
+    flex: 1,
+    alignItems: 'center',
+  },
+  content: {
+    width: '100%',
+    height: '100%',
```

Comment:
if you remove align-items above, you can do this:
```suggestion
    flex: 1,
```